prosody/Dockerfile

```diff
@@ -44,6 +44,7 @@ RUN wget -qO /etc/apt/trusted.gpg.d/prosody.gpg https://prosody.im/files/prosody
       lua-ldap \
       lua-luaossl \
       lua-sec \
+      curl \
```

Comment:
Let's put curl in the base image instead please.

Reply:
done :+1: